classes/page_group_table.php

```diff
@@ -29,12 +29,14 @@ class page_group_table extends \table_sql {
     /** Columns to be displayed. */
     const COLUMNS = [
         'name',
-        'month',
         'fuzzycount',
         'fuzzydurationcounts',
         'fuzzydurationsum',
     ];
 
+    /** @var int The month to display in YYYYMM format (will display current month if null). */
```

Comment:
Based on the setup, is the month ever going to be null? Which might mean the later part of this description may need to change.

Reply:
See above.